meta-facebook/conf/distro/openbmc-fb.conf

```diff
@@ -132,9 +132,9 @@ DEFAULT_TIMEZONE ?= "America/Los_Angeles"
 # Get the OpenBMC verion
 INHERIT += "openbmc_version"
 
-OPENBMC_FB_DEFAULT_FEATURE = "ext2 ipv6 largefile usbgadget usbhost nfs zeroconf"
+OPENBMC_FB_DEFAULT_FEATURE = "ext2 ipv6 largefile usbgadget usbhost nfs zeroconf ipv4"
 OPENBMC_BOARD_FEATURE ?= ""
-DISTRO_FEATURES = "${DISTRO_FEATURES_LIBC} ${OPENBMC_FB_DEFAULT_FEATURE} ${OPENBMC_BOARD_FEATURE}"
```

Comment:
Did this change happen pre-rocko or between rocko and warrior? I am trying to see if we can do it in some backwards compatible way.

Reply:
I think this happened somewhere in between thud-> warrior.

I think having it OPENBMC_FB_DEFAULT_FEATURE works out, we could still leave DISTRO_FEATURES_LIBC for compatibility, just dont rely on it, for example if we are certain we need ipv4,  it wont affect if someone has it on DISTRO_FEATURES_LIBC and you put it on OPENBMC_FB_DEFAULT_FEATURE as well.

Lets just not rely on it coming from default-distrovars.inc